File: API/Entities/Visit.cs
namespace API.Entities
{
    public class Visit
    {
            public Guid Id { get; set; }
            public DateTime VisitDate { get; set; }
            public Guid PatientId { get; set; }
            public User Patient { get; set; }
            public Guid DoctorId { get; set; }
            public Doctor Doctor { get; set; }
            public Guid ClinicId { get; set; }
            public Clinic Clinic { get; set; }
            public bool IsActive { get; set; }
            public decimal? Fee { get; set; }
            public string? Description { get; set; }
            public string? Reccomendations { get; set; }

    }
}

File: client/medplanner-app/src/app/models/visit.ts
import { Clinic } from "./clinic";
import { Doctor } from "./doctor";
import { DoctorAdmissionConditions } from "./doctor-admission";
import { User } from "./user";

export interface Visit {
    id?: string;
    visitDate?: Date;
    patientId?: string;
    patient?: User | null;
    doctorId?: string;
    clinicId?: string;
    doctorAdmissionId?: string;
    clinic: Clinic;
    doctorAdmisison: DoctorAdmissionConditions;
    doctor?: Doctor | null;
    isActive?: boolean;
    description?: string;
    reccomendations?: string;
    fee?: number | null;
}